fractal-puppy/Engine
# Added selector to choose default scene

Inside ConfigurationPanel->Scene a new selector to choose between scenes has been added.

To test:
- Open the engine.
- Create a scene and save it.
- Go to Configuration->Scene and press Refresh Scenes List button.
- Choose the created scene using the "Default Scene" selector.
- Close the engine.
- Open it again and the scene should be loaded.
